lib/zuora/objects.rb

```diff
@@ -27,5 +27,6 @@ module Objects
     autoload :RatePlanData,              'zuora/objects/rate_plan_data'
     autoload :RatePlanChargeData,        'zuora/objects/rate_plan_charge_data'
     # autoload :Refund,                    'zuora/objects/refund'
+    # autoload :RefundInvoicePayment,      'zuora/objects/refund_invoice_payment'
```

Comment:
Any idea why this line is commented out?

Reply:
Reverted in this commit [841f53b](https://github.com/Sage/zuora/pull/18/commits/841f53b823eaa50e77a234490d96a9cee4bdc218)